core/frontend/src/DrawingViewState.ts

```diff
@@ -408,10 +408,10 @@ export class DrawingViewState extends ViewState2d {
     return this._modelLimits;
   }
 
-  /** @internal override */
+  /** @internal */
   public isDrawingView(): this is DrawingViewState { return true; }
```

Comment:
I had been using `@internal override` for two related cases: overriding a base class method, or implementing an abstract base class method.
In both cases the intent is to document that the method is `@internal` intentionally, because the `@public` base class method's documentation suffices.

Reply:
I see... I did not realize that `noImplicitOverride` does not require the `override` keyword for the latter case (and this seems to be [by design](https://github.com/microsoft/TypeScript/issues/44457) - the idea being that if you get the name wrong, you'll still have a compiler error for not implementing the abstract method).

That being said, we still _can_ use the override keyword in these places, so I'm in favor of just doing that for consistency.  I'll push that change, but let me know if you'd rather go back to just mentioning it in the comment.

Reply:
> let me know if you'd rather go back to just mentioning it in the comment.

No - my hope was that we'd switch to using `override` and the scripts we use to detect usage of non-public APIs in apps would be enhanced to ignore usage of methods marked `override` (assuming the base class method is `@public`).